main.py

```diff
@@ -1,12 +1,6 @@
-from typing import Union
-from fastapi import FastAPI 
+from fastapi import FastAPI
+from views import configure_routes
 
 app = FastAPI()
 
-@app.get("/")
-def read_root():
-    return {"Hello": "World"}
-
-@app.get("/items/{item_id}")
-def read_item(item_id: int):
-    return {"item_id": item_id}
\ No newline at end of file
+configure_routes(app)
```

Comment:
I don't feel this function configure_routes is what I was looking for. 
Please have a look at this code snippet below and let me know if need any help
```
from fastapi import APIRouter, FastAPI
from models import Item

app = FastAPI()
router = APIRouter()

@router.trace("/items/{item_id}")
def trace_item(item_id: str):
    return None

app.include_router(router)
```